centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/Widgets/centreon-widget-resourcestable/src/Listing/useListing.ts

```diff
@@ -132,6 +132,32 @@ const useListing = ({
     }
   }, [isOpenTicketEnabled]);
 
+  useEffect(() => {
+    if (isNil(limit) && isNil(sortField)) {
+      setPanelOptions?.({
+        limit: 10,
+        sortField: 'status_severity_code',
+        sortOrder: SortOrder.Desc
+      });
+      return;
+    }
+
+    if (isNil(limit)) {
+      setPanelOptions?.({
+        limit: 10
+      });
+      return;
+    }
+
+    if (isNil(limit)) {
```

Comment:
This condition seems to be unreachable

Reply:
yep you're right, i'll change it right away